bin/firefox-schema-import

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+// vim: set ft=javascript:
```

Comment:
It should, I added a plugin to set it in my vim and it still isn't working though :(

I'll remove this and figure that out on my own though.